pg_shard.c

```diff
@@ -179,6 +184,11 @@ _PG_init(void)
 							 &UseCitusDBSelectLogic, false, PGC_USERSET, 0, NULL,
 							 NULL, NULL);
 
+	DefineCustomBoolVariable("pg_shard.log_distributed_statements",
```

Comment:
Hm, I see. I was under the impression this was only being logged for SELECT's, but looks like its logging even updates and deletes. I'm convinced then to leave it as it is.